usaspending_api/submissions/models.py

```diff
@@ -3,6 +3,8 @@
 
 class SubmissionAttributes(models.Model):
     submission_id = models.AutoField(primary_key=True)
+    db_submission_id = models.IntegerField()
+    usaspending_flag = models.CharField(max_length=1, blank=True, null=True)
```

Comment:
@afrasier Changed usaspending_flag to usaspending_update, which is a date field.